### ISSUE TITLE
How to enable autocomplete in MacOS bash

### ISSUE THREAD

**Issue by hurricaneeb1** [Azure/azure-cli#7829]
As the title explains, I'm searching for a solution to enable the autocomplete feature in MacOS bash.
Now az... isn't autocompleting by using "tab"

### Environment summary

MacOS 10.14.2Beta / CLI version azure-cli (2.0.48)/ bash
```
```

**Comment by tjprescott** [comment]
How did you install the CLI?

**Comment by hurricaneeb1** [comment]
Sorry for that,
I installed the CLI via Homebrew.
I followed this documentation:
https://docs.microsoft.com/de-de/cli/azure/install-azure-cli-macos?view=azure-cli-latest

Thank you :-)

**Comment by yonzhan** [comment]
add to S164.

**Comment by jiasli** [comment]
@fengzhou-msft, could you help verify if `az.completion` is executed correctly on MacOS.

https://github.com/Azure/azure-cli/blob/37b6c8c5d929c7b585a23035744b2efeb62c1db8/scripts/release/homebrew/docker/formula_template.txt#L47

To manually enable auto-complete, please run

```sh
eval "$(register-python-argcomplete az --complete-arguments -o nospace -o default -o bashdefault)"
```

Also see: https://github.com/Azure/azure-cli/issues/6371

**Comment by yonzhan** [comment]
@fengzhou-msft can we close this issue?

**Comment by fengzhou-msft** [comment]
Fixed by #11817, will release with 2.0.81 next week.

**Comment by fengzhou-msft** [comment]
@hurricaneeb1 azure-cli installs az.completion to /usr/local/etc/bash_completion.d/, the folder is created when Homebrew is installed. You need to configure completions for your shell first to enable completions in this directory: https://docs.brew.sh/Shell-Completion
